#!/usr/bin/env python

PACKAGE = "multi_device_view"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("range_th_min", int_t, 0, "The min depth to be detected", 400, 0, 10000)
gen.add("range_th_max", int_t, 0, "The max depth to be detected", 1000, 0, 10000)
exit(gen.generate(PACKAGE, "multi_device_view", "MultiDeviceView"))
